### ISSUE TITLE
Settings: add tab to enable/disable runtime

### ISSUE THREAD

**Issue by plata** [PhoenicisOrg/phoenicis#2180]

**Comment by madoar** [comment]
I think this issue is quite important and should be implemented for the next alpha release. I think it will also help other people who face problems with the current runtime package. I suggest that we enable the runtime by default and allow the user to disable it later on.

**Comment by plata** [comment]
The main problem is: this is a global setting for an engine. So the GUI cannot know that this setting exists.

**Comment by madoar** [comment]
We could add a config file to each engine category where we allow specifying such settings. What do you think?

**Comment by plata** [comment]
Yes, that's a possibility. However, it could be annoying because you would have to switch it off for every single container.
